docs/application/dotnet/index.md

```diff
@@ -28,7 +28,7 @@ Managed runtime offers the following advantages to your application development:
 
 - **Cross-platform support**
 
-  The managed code is portable. Tizen .NET applications can have large portions of their logic applied to other systems supported by the .NET Core and Xamarin.Forms.
+  The managed code is portable. Tizen .NET applications can have large portions of their logic applied to other systems supported by .NET Core and Xamarin.Forms.
```

Comment:
For Figure: "Tizen .NET architecture"

Please verify if it is the latest image.

Reply:
The latest image url is https://d3unf4s5rp9dfh.cloudfront.net/Tizen_doc/cs_overview.png

Reply:
@hakkyu-samsung,

Thank you for the image. I will update.